lib/smart_answer/erb_renderer/format_capture_helper.rb

```diff
@@ -2,60 +2,80 @@ module SmartAnswer
   module ErbRenderer::FormatCaptureHelper
     class InvalidFormatType < RuntimeError; end
 
-    DEFAULT_FORMATS = {
-      govspeak: [/^body$/, /^post_body$/, /^next_steps$/],
-      text: [/^title$/, /^meta_description$/, /^hint$/, /^label$/, /^suffix_label$/, /^error_*./],
-    }.freeze
+    TEXT_CONTENT = [
+      :title,
+      :meta_description,
+      :hint,
+      :label,
+      :suffix_label,
+      /^error_/,
+    ].freeze
```

Comment:
How come we switched to an array of mixed types? Feel this adds more complexity and reduces clarity of how these rules are evaluated.  E.g. what are allow types for this array, what takes precedence a symbol or regex, which one should I use?

Reply:
It seemed easier to parse as non-regexs as you don't have to mentally subtract the `/^` and `$/` from each item to understand the contents. It also seems a relatively common pattern in Ruby to allow a regex for a pattern otherwise an exact match. Precedence doesn't impact as if any match it'll be deemed text only. I wasn't too concerned about the allowed types list since this a private concern of this class.

That said I'm not too fussed about it, just seemed a bummer to make all the names regexes for the sake of one that was a pattern. 

How strongly do you feel about it? prefer them all as regexs?

Reply:
Yup! Sorry you're right, precedence (was my main concern) isn't an issue - was still thinking we are using these rules to distinguish between multiple format types.